packages/grid_client/src/clients/tf-grid/contracts.ts

```diff
@@ -251,7 +259,37 @@ class TFContracts extends Contracts {
       throw err;
     }
   }
+  /**
+   * Get contract discount package
+   * @param {GetDiscountPackageOptions} options
+   * @returns {Promie<DiscountLevel>}
+   */
+  async getDiscountPackage(options: GetDiscountPackageOptions): Promise<DiscountLevel> {
+    const gqlClient = new Graphql(options.graphqlURL);
+
+    const body = `query getConsumption($contractId: BigInt!){
```

Comment:
I notice that we can add order to the request body; i think this will be better 
![image](https://github.com/user-attachments/assets/88c22ce4-019a-4873-b74a-a1643e705bbc)

Reply:
it doesn't change order in  UI but alright added